.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build1:
+    name: build_group1
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ["cassandra", "cgit", "clr-installer-ci", "elasticsearch", "flink", "golang", "haproxy", "httpd", "iperf", "machine-learning-ui", "mariadb", "mixer-ci", "nginx", "node", "openjdk/8", "openjdk/13", "openvino", "os-core", "perl", "php", "php-fpm", "postgres", "python", "rabbitmq", "redis", "ruby", "tensorflow", "tensorflow-serving", "tomcat", "wordpress"]
```

Comment:
You can delete java containers, such as "cassandra", "elasticsearch", "tomcat" and "flink".